FirebaseAppCheck/Sources/AppAttestProvider/Storage/FIRAppAttestKeyIDStorage.m

```diff
@@ -22,11 +22,22 @@
 #import "FBLPromises.h"
 #endif
 
+#import "FirebaseAppCheck/Sources/Core/Errors/FIRAppCheckErrorUtil.h"
+
+/// The `NSUserDefaults` suite name for the storage location of the app attest key ID.
+static NSString *const kFIRAppAttestKeyIDStorageDefaultsSuiteName = @"com.firebase.FIRAppAttestKeyIDStorage";
```

Comment:
Follow-up question: Should "com.firebase.appcheck" be used instead of "com.firebase.FIRAppAttestKeyIDStorage"? I referenced [the conversation in this pull request](https://github.com/firebase/firebase-ios-sdk/pull/7893#discussion_r612891887) and was wondering if "com.firebase.appcheck" is a better name for a suite of appcheck related defaults (and, if so, then  should [`FIRAppCheckSettings` use the "com.firebase.appcheck" suite as well– instead of using the shared user defaults instance](https://github.com/firebase/firebase-ios-sdk/blob/8877fb3554b0e40c7e57e6c078f91067c4343a9f/FirebaseAppCheck/Sources/Core/FIRAppCheck.m#L123)).

Reply:
I think using a class specific suite name for this case is a good idea, so  "com.firebase.FIRAppAttestKeyIDStorage" LGTM.